### ISSUE TITLE
Figure out how to migrate analyzed sets/serials

### ISSUE THREAD

**Issue by ahafele** [sul-dlss/FOLIO-Project-Stanford#141]
related to #140 

examples: https://searchworks.stanford.edu/view/463376 
https://searchworks.stanford.edu/view/1345046 

Partially analyzed (this is something that happens but not as often) - https://searchworks.stanford.edu/view/358930

**Comment by dlrueda** [comment]
Need to pair with Alissa on this - topic for a Libsys “story time.” 
How do we identify analyzed sets in Symphony and prep for export.
How should these look in FOLIO once migrated?
Then we can figure out what needs to be done to get them migrated correctly.

**Comment by dlrueda** [comment]
question: we can’t see a good way in Symphony to identify adequately the right records to link together.
question: are we creating the “dummy” records on the series title now as part of migration? should we stop?

Shelley and Darsi tried to do some identification and think that if a spreadsheet of data is created by a cataloger with the right catkeys of series and individual volumes, then we can create the links during migration.  But identifying them by series title seems problematic.

**Comment by shelleydoljack** [comment]
This consul documentation https://consul.stanford.edu/display/MD/Bound+withs suggests that some analyzed sets have BW-PARENT and BW-CHILD in the item cat 1 fields. These will end up getting the standard bound-with relationship. Is there a desire to also create a different relationship (monographic series, etc) at the instance level?

**Comment by ahafele** [comment]
Per Jeanette - 
Yes, we have four different kinds of multivolume sets: classed as separates (CAS), analyzed separately, analyzed together, and a blend
- CAS is now only used by ordering to keep track of standing orders. The set/serial record is shadowed and the call numbers do not have |z in them on the records. The call number for the set is CALL # VARIES
- Analyzed separately is like this one where only a placeholder item is on the set/serial record and all the volumes have their own record with the set call number and volume information.
- Analyzed together would look exactly the same in the bibliographic record. The only difference is there would be no public note in the MHLD saying the items cataloged separately, and all the items are on the record itself. Sometimes depending on SULs practice serials will have a placeholder item and/or a basecalnum item for the serial.
- Blended ones would have the public note like the analyzed separately ones, but the items would be like the analyzed together.

Basically for all of them the bib information is the same and the difference lies in items and MHLD information.

**Comment by ahafele** [comment]
I propose we don't tackle this before migration since identifying them is so challenging. Review with Vitus first? @dlrueda any open migration questions related to this? e.g. your question from above - are we creating the “dummy” records on the series title now as part of migration? should we stop?

**Comment by ahafele** [comment]
Talk in libsys meeting on friday.

**Comment by ahafele** [comment]
@ahafele will get some examples and make sure we are migrating those locations. Question about dummy records no longer open. Instance level linking can happen by staff in nolana. Additional linking functionality coming in orchid/poppy

**Comment by ahafele** [comment]
Still need to talk with Jeanette about ongoing processes.

**Comment by ahafele** [comment]
Closing since already migrated and Jeanette has been working on ongoing processes. Will open further ticket if needed.
